hazelcast/src/main/java/com/hazelcast/sql/SqlErrorCode.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) 2008-2020, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.sql;
+
+/**
+ * Error codes used in Hazelcast SQL.
+ */
+public final class SqlErrorCode {
```

Comment:
We will have dozens of error codes in the future. They are part of public API and are transferred over the wire. For this reason, it is better to use ints to prevent compatibility problems.